Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -1527,6 +1529,9 @@
   MSFT:*_*_*_CC_FLAGS = /D DISABLE_NEW_DEPRECATED_INTERFACES
   GCC:*_*_*_CC_FLAGS = -D DISABLE_NEW_DEPRECATED_INTERFACES
 
+[BuildOptions.IA32]
+  MSFT:*_*_*_DLINK_FLAGS = /ALIGN:64

```

Comment:
Why is this needed here?

Reply:
This is for the changes in openssl.  We need alignment for using Opensslexcel versions.